### PR TITLE
[COV-703] Ingest Rush Clinical Data 11 Feb 2021

### PR DESCRIPTION
**Pull request by svburke**
Jira Ticket: [PXP-xxxx](https://ctds-planx.atlassian.net/browse/COV-703)

### New Features
Add the reference_file node to house previous versions of data for new data sets.

### CODE REVIEW

**Review by xritter1 — APPROVED** [review]

File: gdcdictionary/schemas/reference_file.yaml [hunk 1]
```diff
@@ -0,0 +1,144 @@
...
+  - md5sum
+  - data_category
+  - data_type
+  - data_format
+
+uniqueKeys:
+  - [ id ]
+  - [ project_id, submitter_id ]
+
+properties:
```
Comment (paulineribeyre) [inline gdcdictionary/schemas/reference_file.yaml:55]: should we add a "vresion" prop, and/or maybe a "description" prop?

Comment (svburke) [reply gdcdictionary/schemas/reference_file.yaml:55]: Created a version property.

Comment (svburke) [reply gdcdictionary/schemas/reference_file.yaml:55]: Created a version property.

**Review by paulineribeyre — APPROVED** [review]
lgtm

**Review by xritter1 — APPROVED** [review]

**Pull request merged**